docs/js/app.js

```
@@ -0,0 +1,185 @@
+                    lastCodeBlock = await addElementToHTML("div", "example-block", "");
+
+                    await addElementToChildHTML(lastCodeBlock, "pre", "example-pre", "");
+                }
+                break;
+            case line.startsWith("---"):
+                tableMode = false;
+                addElementToHTML("hr", "section-separator", "");
+                break;
+            case line.startsWith("-"):
+                tableMode = false;
+                let parsedLine = line.replace("- ", "");
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)